robottelo/vm.py

```diff
@@ -372,6 +372,56 @@ def execute_foreman_scap_client(self, policy_id=None):
             raise VirtualMachineError(
                 'Failed to execute foreman_scap_client run.')
 
+    def configure_rhai_client(self, activation_key, org, rhel_distro):
```

Comment:
@sthirugn moved it to vm.py as all the operations are vm related and RHAI is a part of satellite now. Also, moving it to vm.py gives me more control over the tests, can use the VM context manager easily, no try/catch block, more cleaner and sorted tests. Also, @elyezer and @kbidarkar agreed to the same.